src/CoreWCF.Http/src/CoreWCF/HttpTransportSecurity.cs

```diff
@@ -95,7 +95,7 @@ private void DisableAuthentication(HttpTransportBindingElement http)
         //    return http.AuthenticationScheme == AuthenticationSchemes.Anonymous && http.ProxyAuthenticationScheme == AuthenticationSchemes.Anonymous && http.Realm == DefaultRealm;
         //}
 
-        internal void ConfigureTransportProtectionAndAuthentication(HttpsTransportBindingElement https)
+        public void ConfigureTransportProtectionAndAuthentication(HttpsTransportBindingElement https)
         {
```

Comment:
Copy the implementation as helper methods into the new project as this shouldn't be public.

Reply:
This is addressed in cba7f5ab675f27f254cc71c0675feb4e0bdd2f8b.